backend/todo/migrations/0003_todo_due_date.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0a1 on 2021-10-18 20:52
```

Comment:
no change but it can be nice to combine the migrations into a single migration file if you are adding multiple columns at once. You can recreate by deleting 0002 and 0003 after your realize yo've made 2 files and then regenerating